internal/install/types/errors.go

```diff
@@ -113,3 +186,41 @@ func (e *UncaughtError) Error() string {
 
 // nolint: golint
 var ErrorFetchingLicenseKey = errors.New("Oops, we're having some difficulties fetching your license key. Please try again later, or see our documentation for installing manually https://docs.newrelic.com/docs/using-new-relic/cross-product-functions/install-configure/install-new-relic")
+
+type ErrUnalbeToFetchLicenseKey struct {
+	Err     error
+	Details string
+}
+
+func (e *ErrUnalbeToFetchLicenseKey) Error() string {
+	return "could not fetch license key"
+}
```

Comment:
Typo: `ErrUnalbeToFetchLicenseKey` > `ErrUnableToFetchLicenseKey`

Which brings up another question: is this type referenced anywhere?

Reply:
This is actually existing behavior, I just moved the code